openapi/tests/openapi_integration/test_payload_indexing.py

```diff
@@ -169,10 +176,49 @@ def test_datetime_indexing():
             },
         )
         assert response.ok, response.json()
-        
+
         point_ids = [p["id"] for p in response.json()["result"]["points"]]
         assert all(id in point_ids for id in expected_ids)
 
+    # test with wrong/unsupported datetime format
+    wrong_data = [
+        # RFC 2822 format
+        ({"gte": "Thu, 01 Jan 2015 01:00:00 +0100",
+         "lte": "Thu, 01 Jan 2015 01:00:00 +0100"}, [1]),
+        # Unix millisecond timestamp
+        ({"gte": "1422758400000", "lte": "1422758400000"}, [1]),
+        # Unix second timestamp
+        ({"gte": "1420070400", "lte": "1420070400"}, [1]),
+        # Human readable format
+        ({"gte": "2015-02-01 06:00:00 AM",
+         "lte": "2015-02-01 06:00:00 AM"}, [2]),
+        # Here are some elasticsearch built-in datetime format
+        # Reference: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html#built-in-date-formats
+        # date_optional_time
+        ({"gte": "2015-02-01T06:00:00.000000000",
+         "lte": "2015-02-01T06:00:00.000000000"}, [2]),
+        # basic_date
+        ({"gte": "20150201", "lte": "20150201"}, [2]),
+        # basic_date_time
+        ({"gte": "20150201T060000.000Z", "lte": "20150201T060000.000Z"}, [2]),
+        # basic_ordinal_date_time_no_millis
+        ({"gte": "2015032T060000Z", "lte": "2015032T060000Z"}, [2]),
+        # wrong format mixed with supported format
+        ({"gte": "2015032T060000Z", "lte": "2015-02-01T06:00:00"}, [2]),
+    ]
+    for range_, expected_ids in wrong_data:
+        response = request_with_validation(
+            api="/collections/{collection_name}/points/scroll",
+            method="POST",
+            path_params={"collection_name": collection_name},
+            body={
+                "with_vector": False,
+                "filter": {"must": [{"key": datetime_key, "range": range_}]},
+            },
+        )
+        assert not response.ok
```

Comment:
Ok 👍

Reply:
I don't think API will be returned proper error here, but not because of the incorrect implementation, but because  serde doesn't propagate errors of individual untagged enum variants.

Looks to me that this should be out of scope for this PR.

Reply:
Alright. Let's keep it like this then!